pandas/tests/plotting/common.py

```diff
@@ -53,8 +53,10 @@ def setup_method(self, method):
         import matplotlib as mpl
         mpl.rcdefaults()
 
-        self.mpl_ge_2_0_1 = plotting._compat._mpl_ge_2_0_1()
+        self.mpl_eq_2_0_2 = plotting._compat._mpl_eq_2_0_2()
```

Comment:
I would just bump mpl to 2.1 min to avoid all of these extra things

Reply:
ok, will do, but the special casing for 2.2.0 will still be necessary, unless we bump to 2.2.2

Reply:
cc @tacaswell any thoughts here?